schematools/importer/geojson.py

```diff
@@ -48,15 +49,22 @@ def parse_records(self, file_name, dataset_table, **kwargs):
         """Provide an iterator the reads the NDJSON records"""
         features = read_geojson(file_name)
         main_geometry = dataset_table.main_geometry
+        db_table_name = get_table_name(dataset_table)
+        relation_field_names = [
+            field.name for field in dataset_table.fields if field.relation is not None
+        ]
         for feature in features:
             if main_geometry is not None:
-                wkt = shape(feature[main_geometry]).wkt
+                wkt = shape(feature["geometry"]).wkt
             record = dict(
                 self._clean_value(name, value)
-                for name, value in feature[main_geometry].items()
+                for name, value in feature["geometry"].items()
```

Comment:
Here is a bug, introduced earlier and fixed in a later commit. "geometry" should be "properties".